homeassistant/components/blebox/climate.py

```diff
@@ -81,9 +81,9 @@ def target_temperature(self):
     async def async_set_hvac_mode(self, hvac_mode):
         """Set the climate entity mode."""
         if hvac_mode == HVAC_MODE_HEAT:
-            return await self._feature.async_on()
-
-        return await self._feature.async_off()
+            await self._feature.async_on()
+        else:
```

Comment:
`return` is better since we can outdent below. Flat is better than nested.

Reply:
Sure.